.config/mise/config.toml

```diff
@@ -4,29 +4,29 @@ _.file = [".env", ".env.local"]
 [tools]
 # Basic tools
 node        = { version = "22.15.0", postinstall = "corepack enable" }
-"npm:turbo" = "2.5.2"
+"npm:turbo" = "2.5.3"
 
 # Helper to install tools
-cargo-binstall = "1.12.3"
-uv             = "0.7.4"
+cargo-binstall = "1.12.5"
+uv             = "0.7.5"
 
 # Tools to compile the project
 'cargo:wasm-opt'  = "0.116.1"
 "cargo:wasm-pack" = "0.13.1"
 java              = "24"
-protoc            = "30.2"
+protoc            = "31.0"
 
 # CLI tools
 biome                  = "1.9.5-nightly.ff02a0b"
 "cargo:cargo-hack"     = "0.6.36"
 "cargo:cargo-insta"    = "1.43.1"
 "cargo:cargo-llvm-cov" = "0.6.16"
-"cargo:cargo-nextest"  = "0.9.95"
+"cargo:cargo-nextest"  = "0.9.96"
 just                   = "1.40.0"
-markdownlint-cli2      = "0.17.2"
+markdownlint-cli2      = "0.18.1"
```

Comment:
this will need some updates on the markdown (tagging @vilkinsons as it's markdown content) as there is a new markdown rule.

Reply:
Ah, yes, I'll fix this as part of this PR.